tool/gravity/cli/register.go

```diff
@@ -84,6 +84,9 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.InstallCmd.Force = g.InstallCmd.Flag("force", "Force phase execution").Bool()
 	g.InstallCmd.Resume = g.InstallCmd.Flag("resume", "Resume installation from last failed step").Bool()
 	g.InstallCmd.Manual = g.InstallCmd.Flag("manual", "Manually execute install operation phases").Bool()
+	g.InstallCmd.ExitAfterError = g.InstallCmd.Flag("exit-after-error", "Exit after operation failure without waiting for input").
```

Comment:
Sorry to be pedantic, but I think `--exit-on-failure` sounds better :) I'm fine with either one though.